frontend/src/utils/formatters.ts

```diff
@@ -49,6 +62,45 @@ export function formatServiceArea(serializedServiceArea: string) {
   let { county, zip } = parseSerializedServiceArea(serializedServiceArea)
   return `${capitalizeWords(county)} / ${zip}`
 }
+/**
+ * Converts long string into array of shorter strings, with given maximum width.
+ */
+export function formatLabel(str: string, maxwidth: number) {
```

Comment:
This seems like a very long function, maybe we can add a TODO to lodash it?

Reply:
added TODO